src/components/EventCard/EventCard.tsx

```diff
@@ -80,47 +106,28 @@ const EventCard = forwardRef<HTMLDivElement, EventCardProps>(
           </div>
 
           <div className="flex h-[9rem] flex-col justify-between gap-4">
-            <div className="flex justify-start gap-4">
-              <CalendarDays className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                活動時間
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {startTime}-{endTime}
-              </p>
-            </div>
-            <div className="flex justify-start gap-4">
-              <Users className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                參加人數
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {attendeeCount}
-              </p>
-            </div>
-            <div className="flex justify-start gap-4">
-              <MapPin className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                舉辦位置
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {location}
-              </p>
-            </div>
-            <div className="flex justify-start gap-4">
-              <Building2 className="flex-shrink-0" size={24} />
-              <p className="h-6 w-16 flex-shrink-0 text-base font-normal">
-                主辦單位
-              </p>
-              <p className="flex-grow truncate text-base font-medium">
-                {organizer}
-              </p>
-            </div>
+            <EventCardInfoSection
+              startTime={format(new Date(startTime), 'MM.dd EE ', {
+                locale: zhTW,
+              })}
+              endTime={format(new Date(endTime), ' MM.dd EE', { locale: zhTW })}
```

Comment:
將日期格式改成和 activity page一致

```suggestion
              startTime={format(new Date(startTime), 'MM.dd （EEEEE）', {
                locale: zhTW,
              })}
              endTime={format(new Date(endTime), ' MM.dd （EEEEE）', {
                locale: zhTW,
              })}
```